### ISSUE TITLE
Identical faces for neighbouring gnus-cite levels

### ISSUE THREAD

**Issue by tardigradus** [greduan/emacs-theme-gruvbox#185]
### I'm submitting a:

- [ ] Feature request
- [ X] Bug report

### Bug Report : Describe the problem in your own words

This concerns gruvbox-light-hard.

The faces for gnus-cite-1,  gnus-cite-2,  gnus-cite-3 are identical. The same is true for the group of levels 4, 5, 6, the group 7, 8, 9 and the group 10, 11 

Is this intentional?  I feel that the colours of neighbouring cite levels should be different, even if the total number of colours is limited and there is repetition.

### Please provide a relevant screenshot to illustrate the problem.

![Screenshot from 2022-02-16 10-10-50](https://user-images.githubusercontent.com/6949573/154233025-3a87cb19-2b31-4616-a2d9-e80f4d37d7b7.png)

### Please tell us about your environment.

- OS version: Debian 11

- Emacs version: GNU Emacs 27.1 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.24, cairo version 1.16.0) of 2021-03-28, modified by Debian

    (use `M-x version` then copy the version string from `**Messages**`)

- If your report is about a specific Emacs major or minor mode, please
  let us know how it was installed (MELPA, ELPA, by hand, etc.) and
  the major/minor-mode version number.

**Comment by jasonm23** [comment]
Thanks for the request.

We'll happily review any pull requests to add mode support / fix defects.

**Comment by jasonm23** [comment]
Fixed in 01bcb9ed82cf00be15900fbe9721f62a6d1fa673

**Comment by tardigradus** [comment]
Thanks for this.  I did work out what changes needed to be made myself, but couldn't get the faces to actually change.  When testing your new version just now, I had the same problem until I realised that I had the Emacs daemon running.  So just ending Emacs and restarting wasn't enough - I had to end Emacs, kill the ```emacs --daemon``` process and then restart.  Now I can see and enjoy the changes.
